### PR TITLE
added new routes with new features

### PR DESCRIPTION
**Pull request by neetusuthar90**
## What?
Add new features to add new items in particular user accounts.
## Why?
So that user can saved all password at a place.
## How?
Create some new routes.

##Screenshot?
image

image

### CODE REVIEW

File: app/main/routes.py [hunk 1]
```diff
@@ -17,7 +19,11 @@ def index():
 def register():
     form = RegistrationForm()
     if form.validate_on_submit():
-        user = User(username = form.username.data, email = form.email.data)
+        user = User.query.filter_by(email=form.email.data).first()
+        if(user is not None):
+            flash('Email already exists. Please choose a different email address!!')
+            return redirect(url_for('main.register'))
+        user = User(username = form.username.data, email = form.email.data, password= form.password1.data)
```
Comment (liptanbiswas) [inline app/main/routes.py:26]: Nit, run linter on it by running Format document in vscode

File: app/main/routes.py [hunk 2]
```diff
@@ -33,7 +39,41 @@ def login():
         if user is not None and user.check_password(form.password.data):
             login_user(user)
             next = request.args.get("next")
-            return redirect(next or url_for('main.index'))
+            return redirect(next or url_for('main.new_item'))
```
Comment (liptanbiswas) [inline app/main/routes.py:42]: probably redirect to vault page?

File: instance/app.sqlite
Comment (liptanbiswas) [inline instance/app.sqlite:1]: fix your gitignore and don't commit this file.

File: app/templates/vault.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="vault">
```
Comment (liptanbiswas) [inline app/templates/vault.html:1]: You have all_items.html that why this vault.html?

Comment (neetusuthar90) [reply app/templates/vault.html:1]: All_items is for new items

**Pull request merged**